[DOCS] Reorganize docs for DL Streamer Pipeline Server (#1789)
open-edge-platform/edge-ai-libraries: microservices/dlstreamer-pipeline-server/docs/user-guide/advanced-guide/detailed_usage/camera/genicam.md

> **Known Limitation:**
>

> > - If one observes `Feature not writable` message while working with the GenICam cameras, then reset the device using the camera software or using the reset property of the Generic Plugin. For more information, refer the [README](https://github.com/open-edge-platform/edge-ai-libraries/blob/main/microservices/dlstreamer-pipeline-server/plugins/camera/src-gst-gencamsrc/README.md).

Reviewer: ```suggestion
> - If you get a `Feature not writable` message while working with the GenICam cameras, then reset the device using the camera software or using the reset property of the Generic Plugin. For more information, refer to the [README](https://github.com/open-edge-platform/edge-ai-libraries/blob/main/microservices/dlstreamer-pipeline-server/plugins/camera/src-gst-gencamsrc/README.md).
```
